### PR TITLE
Add configuration file for the Docker build on Jenkins

### PR DESCRIPTION
**Pull request by dalg24**

### CODE REVIEW

File: docker/jenkins/config.dict [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  'image': 'dalg24/dtk-stack',
+  'tag': 'latest',
+  'name': os.getenv("BUILD_TAG"),
```
Comment (Rombur) [inline docker/jenkins/config.dict]: did you forget about import os?

Comment (dalg24) [inline docker/jenkins/config.dict]: no the `import os` is in the Python script `/usr/local/bin/docker_run.py` that is in the Docker slave container

**Comment by dalg24** [comment]
OK to merge

**Comment by Rombur** [comment]
Looks good. You can merge once the tests have passed.

**Comment by dalg24** [comment]
They did...

**Pull request merged**